File: Kattis/C++/Provinces-and-Gold/Provinces-and-Gold.cpp
#include <bits/stdc++.h>

using namespace std;
using i64 = long long;

int main()
{
    i64 g, s, c, p;

    cin >> g >> s >> c;

    p = 3 * g + 2 * s + c;

    if(p >= 8)
    {
        cout << "Province" << " or ";
    }
    else if(p >= 5)
    {
        cout << "Duchy" << " or ";
    }
    else if(p >= 2)
    {
        cout << "Estate" << " or ";
    }
    else if(p < 0)
    {
        cout << "Copper" << endl;
    }
    if(p >= 6)
    {
        cout << "Gold" << endl;
    }
    else if(p >= 3)
    {
        cout << "Silver" << endl;
    }
    else if(p >= 0)
    {
        cout << "Copper" << endl;
    }

    return 0;
}

File: Kattis/Python/Provinces-and-Gold.py

def main():
    g,s,c = input().split()
    g = int(g)
    s = int(s)
    c = int(c)

    p = 3*g + 2*s + c
    if p >= 8:
        print("Province", "or", end=" ")
    elif p >= 5:
        print("Duchy", "or", end=" ")
    elif p >= 2:
        print("Estate", "or", end=" ")
    elif p < 0:
        print("Copper\n")
    if p >= 6:
        print("Gold\n")
    elif p >= 3:
        print("Silver\n")
    elif p >= 0:
        print("Copper\n")

if __name__ == '__main__':
    main()
